microsoft-365/business/add-autopilot-devices-and-profile.md

```diff
@@ -38,52 +38,81 @@ Devices must meet these requirements:
+2. On the **Basics** page, type a **Name** and optional **Description**.
+
+3. If you want all devices in the assigned groups to automatically convert to Autopilot, set **Convert all targeted devices to Autopilot** to **Yes**. All corporate owned, non-Autopilot devices in assigned groups will register with the Autopilot deployment service. Personally owned devices won't be converted to Autopilot. Allow 48 hours for the registration to be processed. When the device is unenrolled and reset, Autopilot will enroll it. After a device is registered in this way, disabling this option or removing the profile assignment won't remove the device from the Autopilot deployment service. You must instead [remove the device directly](https://docs.microsoft.com/mem/autopilot/enrollment-autopilot#delete-autopilot-devices).
+4. Select **Next**.
+5. On the **Out-of-box experience (OOBE)** page, for **Deployment mode**, choose one of these two options:
+    - **User-driven**: Devices with this profile are associated with the user enrolling the device. User credentials are required to enroll the device.
+    - **Self-deploying (preview)**: (requires Windows 10, version 1809 or later) Devices with this profile aren't associated with the user enrolling the device. User credentials aren't required to enroll the device. When a device has no user associated with it, user-based compliance policies don't apply to it. When using self-deploying mode, only compliance policies targeting the device will be applied.
+
+    > [!NOTE]
+    > Options that appear dimmed or shaded are currently not supported by the selected deployment mode.
+
+6. In the **Join to Azure AD as** box, choose **Azure AD joined**.
+7. Configure the following options:
+    - **End-user license agreement (EULA)**: (Windows 10, version 1709 or later) Choose if you want to show the EULA to users.
+    - **Privacy settings**: Choose if you want to show privacy settings to users.
+    >[!IMPORTANT]
+    >The default value for the Diagnostic Data setting varies between Windows versions. For devices running Windows 10, version 1903, the default value is set to Full during the out-of-box experience. For more information, see [Windows Diagnostics Data](https://docs.microsoft.com/windows/privacy/configure-windows-diagnostic-data-in-your-organization) <br>
+ 
+    - **Hide change account options (requires Windows 10, version 1809 or later)**: Choose **Hide** to prevent change account options from displaying on the company sign-in and domain error pages. This option requires [company branding to be configured in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/fundamentals/customize-branding).
```

Comment:
```suggestion
    - **Hide change account options** (requires Windows 10, version 1809 or later): Choose **Hide** to prevent change account options from displaying on the company sign-in and domain error pages. This option requires [company branding to be configured in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/fundamentals/customize-branding).
```